# This message is for detection results in a harvester coordinates
Header header
rasberry_perception/BoundingBox3D[] bounding_boxes
rasberry_perception/SegmentationLabel3D[] instances
string[] class_labels